lib/material/internal/filebrowser/material_filebrowser_internal.flow

```diff
@@ -418,16 +418,14 @@ renderCustomMFileBrowser(manager : MaterialManager, parent : MFocusGroup, captio
 								makeSubscribe(nameB, \n -> validatePartitionName(n, errorText, existsNames)),
 								makeSubscribe(errorText, \t -> nextDistinct(saveEnable, t |> isNone))
 							],
-							MCols([
-								MFixSize(MText(_("Folder name") + ":", []), TFixed(75.0, 20.0)),
-								TFixed(8.0, 0.0),
-								MPad(0.0, -14.0,
+							MBaselineColsA([
+								MText(_("Folder name"), []),
+								MText(":", []),
```

Comment:
Why are _ ("Folder Name") and ":" in different MText? It could be _ ("Folder name:")